management/examples/example.go

```diff
@@ -16,7 +17,7 @@ func Example() {
 	vmSize := "Small"
 	vmImage := "b39f27a8b8c64d52b05eac6a62ebad85__Ubuntu-14_04-LTS-amd64-server-20140724-en-us-30GB"
 	userName := "testuser"
-	userPassword := "Test123"
+	userPassword := ""
```

Comment:
Don't remember when I changed this. Noticed it in final commit. Keeping this as it is for now. :)